TAGradingServer/account/account-rubric.php

```diff
@@ -555,8 +572,32 @@ function display_files($file, &$output, $part, $indent = 1) {
     
     $min_val = (intval($question['gc_max_value']) > 0) ? 0 : intval($question['gc_max_value']);
     $max_val = (intval($question['gc_max_value']) > 0) ? intval($question['gc_max_value']) : 0;
-    
-    $output .= <<<HTML
+    if($question['gc_is_extra_credit'] == true) {
+        $output .= <<<HTML
+    <tr style="background-color: #f9f9f9;">
```

Comment:
How come the row color is declared, then later each cell color is individually declared to be red/green? Just changing the row color could help readability.